lib/ansible/module_utils/facts/hardware/openbsd.py

```diff
@@ -119,13 +112,22 @@ def get_memory_facts(self):
         return memory_facts
 
     def get_uptime_facts(self):
-        uptime_facts = {}
-        uptime_seconds = self.sysctl['kern.boottime']
+        # On openbsd, we need to call it with -n to get this value as an int.
+        sysctl_cmd = self.module.get_bin_path('sysctl')
+        cmd = [sysctl_cmd, '-n', 'kern.boottime']
+
+        rc, out, err = self.module.run_command(cmd)
```

Comment:
not dealing with os/ioerrors possible on execution

Reply:
(As discussed in chat - `run_command()` already catches those, and isn't wrapped in try/except at most other callsites.